crates/sui-cost/tests/snapshots/empirical_transaction_cost__good_snapshot.snap

```diff
@@ -44,8 +44,8 @@ expression: common_costs_actual
     "storage_rebate": 0
   },
   "TransferWholeSuiCoin": {
-    "computation_cost": 148,
-    "storage_cost": 42,
+    "computation_cost": 131,
```

Comment:
@oxade turns out the bug I thought was limited to programmable transactions also affected TransferSui!